README.md

```diff
@@ -1,70 +1,89 @@
+This branch is compatible with the Garden release of Gazebo.
 
-Currently (for Ignition Dome), each Ignition package has a `bazel` branch to maintain the build rules necessary for that package.
+## Initial Setup
 
 To build, we make the assumption that all necessary libraries and dependencies are checked out in a single directory.
-For compatilbility, we use underscores (`_`) to delimit the names, rather than the typical dashes.
 
-The following instructions assume that you are using Ubuntu 18.04
+The following instructions assume that you are using Ubuntu 22.04
 
 ### Installing Bazel
 
-It is best to use the [Bazel Instructions](https://docs.bazel.build/versions/master/install-ubuntu.html) to install for your platform.
-
-For Ubuntu 18.04:
+To use your system package manager, follow the [Bazel Instructions](https://docs.bazel.build/versions/master/install-ubuntu.html) to install for your platform.
 
+For Ubuntu 22.04
 ```
-sudo apt install curl gnupg
-curl https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
-echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
+# Install the bazel repos
+sudo apt install apt-transport-https curl gnupg -y
+curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor >bazel-archive-keyring.gpg
+sudo mv bazel-archive-keyring.gpg /usr/share/keyrings
+echo "deb [arch=amd64 signed-by=/usr/share/keyrings/bazel-archive-keyring.gpg] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
 
+# Install latest stable bazel version
 sudo apt update && sudo apt install bazel
 ```
 
+
+Alternatively, if you intend to work with multiple bazel versions, you can use [Bazelisk](https://github.com/bazelbuild/bazelisk), which automatically picks a good version of bazel given your current working directory.
+
+```
+# Download bazelisk from Github releases
+wget https://github.com/bazelbuild/bazelisk/releases/download/v1.16.0/bazelisk-linux-amd64
+
+# Place it somewhere in your USER path
+mv bazelisk-linux-amd64 ~/.local/bin/bazel
+chmod +x ~/.local/bin/bazel
+
+# OR place it in your system path
+/usr/local/bin/bazel
+chmod +x /usr/local/bin/bazel
+```
+
 ### Setting up the workspace
 
-To begin with, make a directory to contain all of the Ignition packages and dependencies:
+To begin with, make a directory to contain the workspace
 
 ```
-mkdir ~/ignition/
-cd ~/ignition/
+mkdir ~/gz/
+cd ~/gz/
 ```
 
-Then clone each of the Ignition packages.
-For convenience, we have provided a `.repos` file that can be used by [vcstool](https://github.com/dirk-thomas/vcstool)
+Then clone each of the Gazebo packages.  For convenience, we have provided a `.repos` file containing the relevant branches for each package.
 
 ```
-wget https://raw.githubusercontent.com/ignitionrobotics/ign-bazel/master/example/bazel.repos
+wget https://raw.githubusercontent.com/gazebosim/gz-bazel/garden/example/bazel.repos
```

Comment:
What file should I use? This link doesn't work, and I tried https://github.com/gazebosim/gz-bazel/pull/43/files#diff-b3c5fb2260c8de45c8de2a1b0170ffd91e9136b9e81a774595ce0be17dd1a374, but it's missing the `gz-bzel` repo and when I try to include it I get the following build errors:

```
ERROR: /home/nkoenig/gz/msgs/BUILD.bazel:94:17: no such package '@com_google_protobuf//': The repository '@com_google_protobuf' could not be resolved: Repository '@com_google_protobuf' is not defined and referenced by '//msgs:gzmsgs_cc_proto'
ERROR: /home/nkoenig/gz/msgs/BUILD.bazel:94:17: every rule of type _gz_msgs_generate_gen implicitly depends upon the target '@com_google_protobuf//:protoc', but this target could not be found because of: no such package '@com_google_protobuf//': The repository '@com_google_protobuf' could not be resolved: Repository '@com_google_protobuf' is not defined
ERROR: Analysis of target '//msgs:gzmsgs_cc_proto' failed; build aborted: Analysis failed
```

Reply:
Ah, there is probably a bit of a chicken-and-egg problem here.

For now, I am cloning the `gz-bazel` repo separately to get the right branch, since it is the one changing the most frequently.  CI also assumes that `gz-bazel` is checked out separately (because actions will always do that) and then copied into the workspace.

This PR (and the sequence that follows it for each package) is intended to replace #43, that one can be ignored.

For now, do 

```
cd ~/gz
git clone https://github.com/gazebosim/gz-bazel bazel -b mjcarroll/init_garden
```

Reply:
Additionally, this PR is only setting up the CI and the initial stuff we need to build the `bazel` package.  The ones on top actually build up the gazebo packages.